spec/jobs/sys/site_export/export_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+require 'rake'
+
+describe Sys::SiteExportJob, dbscope: :example do
+  let(:site) { cms_site }
+  let(:job) { Sys::SiteExportJob.new }
+  let(:zip) { job.instance_variable_get(:@output_zip) }
+
+  before do
+    task = OpenStruct.new(source_site_id: site.id)
+    def task.log(msg); puts(msg); end
+    job.task = task
```

Comment:
__[rubocop]__
* Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->